Dockerfile

```diff
@@ -132,12 +132,22 @@ RUN mv target/$(/docker-arch-to-rust-target.sh "${TARGETPLATFORM}")/release/mas-
 
 ## Runtime stage, debug variant ##
 FROM --platform=${TARGETPLATFORM} gcr.io/distroless/cc-debian${DEBIAN_VERSION}:debug-nonroot AS debug
+
+# Inject a wasmtime config which disables cache to avoid issues running with a read-only root filesystem
+ENV XDG_CONFIG_HOME=/etc
+COPY ./misc/wasmtime-config.toml /etc/wasmtime/config.toml
+
```

Comment:
We could also push the config to `/home/nonroot/.config/wasmtime` instead and not set the `XDG_CONFIG_HOME`, but at least here it works even if it runs under another user